src/Microsoft.ML/Data/TextLoader.cs

```diff
@@ -0,0 +1,180 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using System;
+using System.Linq;
+using System.Reflection;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.ML.Data
+{
+    public sealed partial class TextLoaderRange
+    {
+        public TextLoaderRange()
+        {
+        }
+
+        /// <summary>
+        /// Convenience constructor for the scalar case.
+        /// Min and Max are set to the single value ordinal.
```

Comment:
>Min [](start = 12, length = 3)

`<see`, also `ordinal` should be in a `<paramref`.

Reply:
@TomFinley These rules could be enforced as errors too. Though the problem currently is that many of the interfaces aren't commented. I'm not sure if going in stages could be a viable alternative.

A good take on commenting: https://github.com/dotnet/orleans/issues/209#issuecomment-77937822.

Reply:
Thanks, Veikko.

---
In reply to: [189102883](https://github.com/dotnet/machinelearning/pull/142#discussion_r189102883) [](ancestors = 189102883)

Reply:
Thanks for adding the `<paramref`. Min and max still not in `<see`

---
In reply to: [189082232](https://github.com/dotnet/machinelearning/pull/142#discussion_r189082232) [](ancestors = 189082232)